openfecli/commands/quickrun.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import click
+from openfecli import OFECommandPlugin
+import json
+
+
+@click.command(
+    'quickrun',
+    short_help="Run a given transformation, saved as a JSON file"
+)
+@click.argument('transformation', type=click.File(mode='r'),
+                required=True)
+def quickrun(transformation):
+    """Run the transformation (edge) in the given JSON file in serial.
+
+    To save a transformation as JSON, use the following Python recipe (after
+    creating the transformation):
+
+    \b
+        import json
+        with open(filename, 'w') as f:
+            json.dump(transformation.to_dict(), f)
+    """
+    import gufe
+    from gufe.protocols.protocoldag import execute
+    print("Loading file...")
+    dct = json.load(transformation)
+    trans = gufe.Transformation.from_dict(dct)
+    print("Planning the campaign...")
+    dag = trans.create()
+    print("Running the campaign...")
+    dagresult = execute(dag)
```

Comment:
We should have a way to pass `shared` here? That way the files aren't temporary?

Reply:
Added `-d` / `--directory` option. Can rename to `--shared` if that makes more sense.

Still need to test that; it didn't seem used in a manual test with the `DummyProtocol`, but that may be the protocol.